packages/gafl-webapp-service/src/pages/summary/contact-summary/route.js

```diff
@@ -198,19 +198,26 @@ const getLicenseeDetailsSummaryRows = (permission, countryName, request) => {
 const getData = async request => {
   const status = await request.cache().helpers.status.getCurrentPermission()
   const permission = await request.cache().helpers.transaction.getCurrentPermission()
+  const mssgs = request.i18n.getCatalog()
 
   checkNavigation(status, permission)
 
   status.fromSummary = CONTACT_SUMMARY_SEEN
   await request.cache().helpers.status.setCurrentPermission(status)
   const countryName = await countries.nameFromCode(permission.licensee.countryCode)
 
+  let changeLicenceDetails = mssgs.change_licence_details_other
+
+  if (permission.isLicenceForYou) {
+    changeLicenceDetails = mssgs.change_licence_details_you
+  }
```

Comment:
As a rule, never use `let`, use `const`. If `const` could be one of two values, generally use the ternary operator to assign a value. If it's more than that, extract the logic to a new function. It's pretty simple logic here, so a ternary is a good fit: `const changeLicenceDetails = permission.isLicenceForYou ? mssgs.change_licence_details_you : mssgs.change_licence_details_other`

The reason for not using `let` is because `const` doesn't allow you to change the value so it makes code easier to read. It's assigned once and then you always know what it is. Whereas with let, you have to check a full code block and do a let of mental parsing to check what its value is. A fuller discussion is available here: https://medium.com/dailyjs/use-const-and-make-your-javascript-code-better-aac4f3786ca1